File: src/NotesExamensSeancesBundle/Entity/Note.php
<?php

namespace NotesExamensSeancesBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Note
 *
 * @ORM\Table(name="note")
 * @ORM\Entity(repositoryClass="NotesExamensSeancesBundle\Repository\NoteRepository")
 */
class Note
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @var int
     * @ORM\ManyToOne(targetEntity="\UtilisateursBundle\Entity\User")
     * @ORM\JoinColumn(name="etudiant",referencedColumnName="id")
     */
    private $etudiant;

    /**
     * @var int
     * @ORM\ManyToOne(targetEntity="Examen")
     * @ORM\JoinColumn(name="examen",referencedColumnName="id")
     */
    private $examen;

    /**
     * @return string
     */
    public function getNote()
    {
        return $this->note;
    }

    /**
     * @param string $note
     */
    public function setNote($note)
    {
        $this->note = $note;
    }

    /**
     * @var string
     *
     * @ORM\Column(name="note", type="integer")
     */
    private $note;

    /**
     * Get id
     *
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set etudiant
     *
     * @param integer $etudiant
     *
     * @return Note
     */
    public function setEtudiant($etudiant)
    {
        $this->etudiant = $etudiant;

        return $this;
    }

    /**
     * Get etudiant
     *
     * @return int
     */
    public function getEtudiant()
    {
        return $this->etudiant;
    }

    /**
     * Set examen
     *
     * @param integer $examen
     *
     * @return Note
     */
    public function setExamen($examen)
    {
        $this->examen = $examen;

        return $this;
    }

    /**
     * Get examen
     *
     * @return int
     */
    public function getExamen()
    {
        return $this->examen;
    }
}

File: skander-mobile/src/entities/Note.java


package entities;


public class Note {

    private int id ;
    private int etudiant;
    private String etudiantName;
    private int examen;
    private String examenName;
    private float note;

    public String getEtudiantName() {
        return etudiantName;
    }

    public void setEtudiantName(String etudiantName) {
        this.etudiantName = etudiantName;
    }

    public String getExamenName() {
        return examenName;
    }

    public void setExamenName(String examenName) {
        this.examenName = examenName;
    }

    public Note(int id, int etudiant, int examen, float note) {
        this.id = id;
        this.etudiant = etudiant;
        this.examen = examen;
        this.note = note;
      /*   
        ExamenServices eS = new ExamenServices();
        this.etudiantName = eS.getEtudiantName(this.etudiant);
        this.examenName = eS.getExamenName(this.examen);*/
    }

    public Note(int etudiant, int examen, float note) {
        this.etudiant = etudiant;
        this.examen = examen;
        this.note = note;
    }

     

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getEtudiant() {
        return etudiant;
    }

    public void setEtudiant(int etudiant) {
        this.etudiant = etudiant;
    }

    public int getExamen() {
        return examen;
    }

    public void setExamen(int examen) {
        this.examen = examen;
    }

    public float getNote() {
        return note;
    }

    public void setNote(float note) {
        this.note = note;
    }

    @Override
    public String toString() {
        return "Note{" + "id=" + id + ", etudiant=" + etudiant + ", examen=" + examen + ", note=" + note + '}';
    }
    
    
    
    
}
